### DISCUSSION TITLE
What ChromeOS Version Are You On?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1204] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
For me its 129

**Comment by doyourclanwars** [comment]
u didnt include 129

**Comment by CGRmags** [comment]
shut up

On Mon, Oct 28, 2024 at 8:14 AM Player 529 ***@***.***> wrote:

> u didnt include 129
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/1204#discussioncomment-11076048>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBBIJI322IN37B26QOLZ5ZIFHAVCNFSM6AAAAABQPICTNOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBXGYYDIOA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
